docker-compose.yaml

```diff
@@ -8,4 +8,4 @@ services:
     image: nxsearch
     volumes:
       - ./data:/data
-      - ./src:/src
+      - ./src:/build
```

Comment:
Why?

Reply:
because of this: https://github.com/rmind/nxsearch/blob/main/compose/nxsearch.Dockerfile#L15
I think Ive dropped last time